e2e/davinci-app/components/password.ts

```diff
@@ -0,0 +1,23 @@
+import { SingleValueCollector } from '../../packages/davinci/types.ts';
```

Comment:
why do we need the .ts file extensions? Are we not exporting an index file from the davinci package?

Reply:
I'll take a look. I think this would auto-added.